lib/manager/actions/visibilityFilter.js

```diff
@@ -6,6 +6,14 @@ export const setVisibilitySearchText = createAction(
   'SET_PROJECT_VISIBILITY_SEARCH_TEXT',
   (payload: null | string) => payload
 )
+export const setVisibilityLabel = createAction(
+  'SET_PROJECT_VISIBILITY_LABEL',
+  (payload: Array<any>) => payload
```

Comment:
Please use a more specific type than `any`.

Reply:
Took me a while to figure out what broke when I fixed this but got it all working in a1bc3e9